swagger_documentation/swagger.json

```diff
@@ -10305,6 +10305,11 @@
                     "type": "string",
                     "description": "Token for file upload",
                     "example": "9dfc5fe3-d617-4d93-ac11-8afe7e984a4b"
+                },
+                "is_public": {
+                    "type": "boolean",
+                    "description": "True if the file is public",
```

Comment:
can you please change this to "True if the file is attached to a public item version"?

Reply:
That is not actually true.
It can be attached to a public item but the file can remain under embargo so it would be private.